docker-compose.yml

```diff
@@ -21,42 +21,16 @@ services:
       - 9092:9092
       - 29092:29092
 
-  server:
+  deephaven:
     image: redpanda-dxfeed-financial-data/dx-server
     environment:
-      JAVA_TOOL_OPTIONS: -Xmx6g -Ddeephaven.console.type=python -Ddeephaven.application.dir=/data/app.d
+      - JAVA_OPTS=-Xmx12g -Ddeephaven.application.dir=/data/app.d
```

Comment:
We'll be updating other documents soon, but we should migrate to using `START_OPTS` in the majority of our examples instead of `JAVA_OPTS`.